examples/helloworld/helloworld/api_resources.proto

```diff
@@ -0,0 +1,102 @@
+
+syntax = "proto3";
+
+package yelp;
+
+service Yelp {
+  rpc CreateUser (CreateUserRequest) returns (CreateUserReply) {}
+  rpc CreateBusiness (CreateBusinessRequest) returns (CreateBusinessReply) {}
+
+  rpc UpdateUser (UpdateUserRequest) returns (UpdateUserReply) {}
+  rpc UpdateBusiness (UpdateBusinessRequest) returns (UpdateBusinessReply) {}
+
+  rpc GetUser (GetUserRequest) returns (GetUserReply) {}
+  rpc GetBusiness (GetBusinessRequest) returns (GetBusinessReply) {}
+
+  rpc DeleteUser (DeleteUserRequest) returns (DeleteUserReply) {}
+  rpc DeleteBusiness (DeleteBusinessRequest) returns (DeleteBusinessReply) {}
+}
+
+message User {
+  string id = 1;
+  string headshot = 2;
+  repeated Review reviews = 3;
+  repeated Business bookmarks = 4;
+  Metadata metadata = 5;
+}
+
+message CreateUserRequest {
+  User user = 1;
+}
+
+message UpdateUserRequest {
+  User user = 1;
+}
+
+message GetUserRequest {
+  string user_id = 1;
+}
+
+message DeleteUserRequest {
+  string user_id = 1;
+}
+
+message CreateUserReply {
+  User user = 1;
+}
+
+message UpdateUserReply {
+  User user = 1;
+}
+
+message GetUserReply {
+  string user_id = 1;
+}
+
+message DeleteUserReply {
+  string user_id = 1;
+}
+
+
+message Business {
+  string id = 1;
```

Comment:
How about opening hours? Please refer to https://developers.google.com/protocol-buffers/docs/proto3 and implement it.